docs/usage/migration.rst

```diff
@@ -0,0 +1,31 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015-2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+
+[WIP] Data Migration
+--------------------
+
+:code:`Locations` are used to represent different storage systems and possibly
+different geographical locations. :code:`Buckets` but also :code:`Objects` are
+assigned a Location. This architecture provides extra flexibility when we want
+to migrate our data.
+
+When we create a bucket we need to provide a Location, otherwise the default
+one is used.
+
```

Comment:
Objects are not directly assigned to locations, right? They belongs to a bucket which has a Location.

Reply:
From what I understood, they actually do! Let's look into that.